database/models/Product.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+
+const productSchema = new mongoose.Schema({
+    id: Number,
+    productName:String,
+    productPrice: Number,
+    productImageURL: String,
+    dateCreated: Date,
+    dateModified: Date,
+    categoryName: String
+});
+
+const Product = mongoose.model('product', productSchema);
```

Comment:
My model name is the culprit of all my headache, I had productInfo and all the collections are in productInfos and I had to rename it to product to make it more sensible so mongodb adds an "s" at the end for collections